shard-operator-parent/shard-operator-v1/src/test/java/com/redhat/service/smartevents/shard/operator/v1/BridgeIngressServiceTest.java

```diff
@@ -96,6 +97,7 @@ public void setup() {
         when(templateProvider.loadBridgeIngressBrokerTemplate(any(), any())).thenCallRealMethod();
         when(templateProvider.loadBridgeIngressKubernetesIngressTemplate(any(), any())).thenCallRealMethod();
         when(templateProvider.loadBridgeIngressOpenshiftRouteTemplate(any(), any())).thenCallRealMethod();
+        doCallRealMethod().when(templateProvider).updateMetadata(any(), any(), any());
```

Comment:
It's peculiar how your PR now needs this in `v1` but it was not needed previously.

I have no idea why :smile:

Reply:
@manstis it was promoted from private to public :D

Reply:
also see https://issues.redhat.com/browse/MGDOBR-1370?filter=-2